README.md

```diff
@@ -59,7 +59,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+<!-- The first expression creates the variable 'weather'. The second expression assigns the string 'sunny' to the variable weather. The last expression is one of boolean equality and evaluates to true because the variable weather is 'sunny'. -->
```

Comment:
These statements are true, but they don't explain what each of those lines evaluates to. 

The first line, `var weather;`, evaluates to `undefined`.
The second line, `weather = "sunny"`, evaluates to `"sunny"` because the `=` operator always evaluates to the value that it's assigning.
The third line, `weather === "sunny"`, evaluates to `true`.